### ISSUE TITLE
New concept for nutrium/food consumption

### ISSUE THREAD

**Issue by Alteraner** [StrangeLoopGames/EcoSuggestions#759]
In later state of the Game, most of the work is done by machines (e.g. Skid, Excavator,...) and the player has learned/used all stars and there is no further need to consume better food. The profession farming and cocking is getting more and more obsolete. Following idea should keep the usage of food up during the game.
First of all, all food is getting an Tier level. Means, food which is for example simple and can be cooked on the campfire is Tier 1, while food cooked on the stove or better could be Tier 4. Depends on the level which is needed to craft the food. In general the idea is, that food with Tier level can only fill the stomach up to a specific level. This means, your stomach divided into different levels. These levels are based on the amount of stars you have used so far. Let me explain it on an example. If you have used one star so far, you can use Tier 1 to Tier 4 food to fill your stomach up to 100%. If you have used two stars, Tier 1 food will only fill your stomach up to 50%. To go to 100% you need to eat Food with Tier 2 till 4. Further if you have used 3 stars you can fill your stomach up to 25% with Tier 1 food, up to 50% with Tier 2 food and up to 100% with Tier 3 and 4 food. This will continue with each star more. Attached graphic should illustrate the behavior. At least i would although add to the game, that using machines/tables and vehicles like Skid consumes food. Not that much like i would do handwork, but still consume food. Today i can work up to three days without the need to eat anything. 
The benefits of this concept is, as mentioned the need for good food is getting more important than just for skill rate. Farmer and chef still needed in mid and endgame phase. Because the number of stars i have used represent as well my personal evolution and my needs for better food increase. Further the concept although prevent users not able to do something in case no good food is available. They still can do work with Tier 1 food but need to eat a lot more than eat better Tier food.

![image](https://user-images.githubusercontent.com/47274045/54905536-91121400-4ee1-11e9-8c05-9ed2fde5f4d3.png)

Its an extension of sugesstion #604

**Comment by Guedez** [comment]
But why would you even care if your stomach is full at all? This does not address the root problem
Also, I can easily mass produce 500 papaya and 250 charred meat, so needing more food is also moot

**Comment by Alteraner** [comment]
Yes, but if you have to eat something after 2 or three times used any tool, it will run you creazy. No normal person will do this and be forced to have better tier food in to have a full stomach.
